packages/backend/src/scheduler/SchedulerWorkerEventEmitter.ts

```diff
@@ -81,14 +81,6 @@ schedulerWorkerEventEmitter.on('job:error', ({ worker, job, error }) => {
 schedulerWorkerEventEmitter.on('job:failed', ({ worker, job, error }) => {
     const message = `Worker ${worker.workerId} failed job ${job.id} (${job.task_identifier}). ${error}`;
     Logger.info(message);
-    Sentry.captureException(new Error(message), {
```

Comment:
this was unnecessary, because the exception is being reported already in SchedulerTaskTracer.